features/projects/components/project-list/project-list.tsx

```diff
@@ -27,7 +39,17 @@ export function ProjectList() {
 
   if (isError) {
     console.error(error);
-    return <div>Error: {error.message}</div>;
+    return (
+      <Badge color={BadgeColor.error} size={BadgeSize.lg}>
+        <RiErrorWarningLine style={{ padding: 3 }} />{" "}
+        <p style={{ padding: 40 }}>
+          There was a problem while loading the project data.
+        </p>
+        <Reload href="http://localhost:3000/projects" className="btn">
+          Try Again <AiOutlineArrowRight style={{ padding: 5 }} />
```

Comment:
I wouldn't mix inline styles with styled-components if not necessary (e.g. when using a JS animation library)